script/build_frontend

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+# Builds the frontend for production
+
+# Stop on errors
+set -e
+
+cd "$(dirname "$0")/.."
+
+rm -rf final
+cp -r public final
+
+# Build frontend
+BUILD_DEV=0 ./node_modules/.bin/gulp
+cp bower_components/webcomponentsjs/webcomponents-lite.js final
+cp bower_components/webcomponentsjs/custom-elements-es5-adapter.js final
+cp build/*.js build/*.html final
+mkdir final/panels
+cp build/panels/*.html final/panels
+
+script/update_mdi.py
```

Comment:
Should this run as part of each build?

Reply:
Yes, it needs to be part of the output that we generate for the PyPi package.

You don't run this script when you're developing locally.

Reply:
Local development workflow will be like this:

 - Once: run `script/build_frontend`
 - On each dev cycle, just use `yarn dev-watch` unless you're going to change files in the `public` directory.